common/elasticsearch/mapping.go

```diff
@@ -181,6 +181,10 @@ var mapping = `{
     }
   }
 },
+"Cluster": {
```

Comment:
you should add this tag under "customMetricTypeSchema()" and not specifically for the general family